File: src/ast/LiteralOperator.java
package ast;

import java.io.StringWriter;

public class LiteralOperator extends ParentNode {
    public LiteralOperator(BaseNode child) {
        super(NodeType.LiteralOperator, child);
    }

    @Override
    public void PrintLeft(StringWriter writer) {
        writer.write("operator \"");
        child.PrintLeft(writer);
        writer.write("\"");
    }
}

File: src/de/fearlesstobi/demangler/ast/LiteralOperator.kt
package de.fearlesstobi.demangler.ast

import java.io.StringWriter

class LiteralOperator(child: BaseNode?) : ParentNode(NodeType.LiteralOperator, child) {
    public override fun printLeft(writer: StringWriter) {
        writer.write("operator \"")
        child!!.printLeft(writer)
        writer.write("\"")
    }
}